deepchem/models/tensorgraph/layers.py

```diff
@@ -1398,6 +1400,363 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     return out_tensor
 
 
+class LSTMStep(Layer):
+  """ LSTM whose call is a single step in the LSTM.
+
+  We generate a sequence of inputs using the intermediate outputs of the
+  LSTM, and so allow step by step operation of the lstm
+  """
+
+  def __init__(self,
+               output_dim,
+               input_dim,
+               init='glorot_uniform',
+               inner_init='orthogonal',
+               activation='tanh',
+               inner_activation='hard_sigmoid',
+               **kwargs):
```

Comment:
This API is inconsistent with other layers, both in terms of the argument names and in passing strings rather than functions.  It would be better to specify `weights_initializer=tf.glorot_uniform_initializer`, `activation_fn=tf.tanh`, etc.